### DISCUSSION TITLE
Can I get information of each sample in a project?

### DISCUSSION THREAD

**Discussion by sqs1020** [GreenleafLab/ArchR#1265] [Questions / Documentation]
Hi,

Is there a way to see clusters of each sample/group from an ArchR project like "split.by" option in Seurat (see attached scrreenshot)? 
PA.p3 <- DimPlot(PA.combined.sct, reduction = "umap", label = TRUE, repel = TRUE, split.by = "group")
Idents(PA.combined.sct) = "orig.ident"
PA.p4 <- DimPlot(PA.combined.sct, reduction = "umap", repel = TRUE, split.by = "group")
![image](https://user-images.githubusercontent.com/52260392/150452038-5ae9f84e-6a5c-4dfe-a117-0daff46c74b9.png)

Thanks!

**Comment by rcorces** [comment]
perhaps you want the `highlightCells` argument?
<html>
<body>


highlightCells | A character vector of cellNames describing which cells to hightlight if using plotAs = "points" (default if discrete). The remainder of cells will be colored light gray.
-- | --



</body>
</html>

https://www.archrproject.com/reference/plotEmbedding.html

**Reply by sqs1020** [reply]
Thank you for the infromation. I saw the highlightCells argument. The cells should in Embedding CellNames. Does that mean I could I only see 1 cell per time? How can I highlight a group of cells at one time? Thank you!
![image](https://user-images.githubusercontent.com/52260392/151997947-d1c1942a-a774-461c-906e-0a2f08831762.png)

![image](https://user-images.githubusercontent.com/52260392/151998155-948be625-ae88-4fa6-b242-53697694e4f7.png)

**Reply by rcorces** [reply]
The function documentation is very explicit. And you have not provided the `highlightCells` argument the correct input. It wants a character vector of cellNames.

<html>
<body>


highlightCells | A character vector of cellNames describing which cells to hightlight if using plotAs = "points" (default if discrete). The remainder of cells will be colored light gray.
-- | --



</body>
</html>
